metrics/src/main/proto/collector.proto

```diff
@@ -0,0 +1,116 @@
+syntax = "proto3";
```

Comment:
Is there any reason to not use the generated files? i.e. lightstep-tracer-java-common/common/src/main/java/com/lightstep/tracer/grpc/SpanContext.java

Reply:
The `collector.proto` file is in this project because of [this line](https://github.com/lightstep/lightstep-tracer-java-common/blob/0791b0e565232ea694a3d8c54dbad9b15c444dbc/metrics/src/main/proto/metrics.proto#L10).